src/Paprika/Merkle/ComputeMerkleBehavior.cs

```diff
@@ -198,7 +260,62 @@ public Keccak BeforeCommit(ICommitWithStats commit, CacheBudget budget)
 
             RootHash = rootKeccak.Keccak;
 
-            return rootKeccak.Keccak;
+            return RootHash;
+        }
+    }
+
+    public void RecalculateStorageTries(ICommitWithStats commit, CacheBudget budget)
+    {
+        using (_storageProcessing.Measure())
```

Comment:
Is this measurement useful here? It's used for regular node running so that we can attach with `dotnet counters` I'd remove.

Reply:
This method was removed, but removed I removed gathering these metrics for `RecalculateStorageTrie`. This can be useful for measuring sync performance.